Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/DeployIISWebApp.ps1

```diff
@@ -54,6 +54,8 @@ function Get-ScriptToRun
     )
```

Comment:
For IIS we didn't increase intentionally as we have took a contribution on the same day with log message related changes